app/code/Mastering/SampleModule/Api/ItemRepositoryInterface.php

```diff
@@ -13,5 +13,24 @@ interface ItemRepositoryInterface{
     /**
      * @return \Mastering\SampleModule\Api\Data\ItemInterface[]
      */
-    public function getList();
+    public function getList(\Magento\Framework\Api\SearchCriteriaInterface $searchCriteria);
+
+    /**
+     * Create or update a data
+     */
+
+    public function save(\Mastering\SampleModule\Api\Data\ItemInterface $item);
```

Comment:
add just methods you need to implement, you don't need to add all CRUD methods when you need to use only one